include/NovelRT/Ecs/Configurator.h

```diff
@@ -28,22 +28,28 @@ namespace NovelRT::Ecs
 
         inline void AddDefaultComponentsAndSystems(SystemScheduler& target)
         {
-            target.GetComponentCache().RegisterComponentType(Graphics::RenderComponent{0, 0, 0, 0, true});
+            target.GetComponentCache().RegisterComponentType(Graphics::RenderComponent{0, 0, 0, 0, true},
+                                                             "NovelRT::Ecs::Graphics::RenderComponent");
```

Comment:
Is this naming convention a requirement? Or can the user use any unique string?

Reply:
It can use any unique string - namespaces just made the most sense to my brain.